src/amo/components/GuidesAddonCard/styles.scss

```diff
@@ -0,0 +1,99 @@
+@import '~amo/css/styles';
+
+.GuidesAddonCard {
+  display: flex;
+  flex-direction: column;
+
+  // stylelint-disable max-line-length
+  // TODO: check on get-firefox button style.
+  // See https://github.com/mozilla/addons-frontend/issues/6432#issuecomment-433083079.
+  .Button--get-firefox,
+  .AMInstallButton {
+    @include respond-to(medium) {
+      align-self: center;
+      margin-left: auto;
```

Comment:
Is this overriding an inherited style? If it needs RTL support then you'd need to use `@include margin-start(auto)`.